ansys/api/additive/v0/additive_operations.proto

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 ANSYS, Inc.
+
+syntax = "proto3";
+
+package ansys.api.additive.v0;
+
+option csharp_namespace = "Ansys.Api.Additive";
+
+import "google/protobuf/timestamp.proto";
+
+// Metadata for a long running operation
+message OperationMetadata {
+    google.protobuf.Timestamp CreateTime = 1;
+    google.protobuf.Timestamp EndTime = 2;
+    string simulationId = 3;
+    string context = 4;
+    string message = 5;
+    float percentComplete = 6;

```

Comment:
```suggestion
message OperationMetadata {
    google.protobuf.Timestamp creation_time = 1;
    google.protobuf.Timestamp end_time= 2;
    string simulation_id = 3;
    string context = 4;
    string message = 5;
    float percent_complete = 6;
```

For better or worse, I used snake case when defining my messages previously. I'm just wanting to keep things consistent.

Reply:
Changed to snake case